src/gc/env/gcenv.structs.h

```diff
@@ -17,15 +17,11 @@ struct GCSystemInfo
 typedef void * HANDLE;
 
 #ifdef PLATFORM_UNIX
-
-typedef char TCHAR;
-#define _T(s) s
-
+typedef char16_t WCHAR;
 #else
 
 #ifndef _INC_WINDOWS
-typedef wchar_t TCHAR;
```

Comment:
These changes were introduced in 0462be1dde80
Since then none of usages of TCHAR have survived to this day. So I'm renaming `TCHAR` to `WCHAR` and replacing `char` with `char16_t` for `PLATFORM_UNIX`.